util/testharness.cc

```diff
@@ -29,6 +30,19 @@ std::string TmpDir(Env* env) {
   return dir;
 }
 
+std::string PerThreadDBPath(std::string dir, std::string name) {
```

Comment:
Some of our tests are multi-threaded in which multiple threads access the same db directory. The name `PerThreadDBPath` can be misleading in such cases, causing people to think that each of these threads have access to a private db directory. Maybe rename it to `PerTestDBPath`?

Reply:
Well if they do call this function from different threads they will get different db path. They however call this in the main thread, and share the db among all the created threads. I think in context it is clear.